srcpkgs/hashboot/template

```diff
@@ -1,18 +1,26 @@
 # Template file for 'hashboot'
 pkgname=hashboot
-version=0.9.8
+version=0.9.9
 revision=1
+hostmakedepends="asciidoc"
 short_desc="Check integrity of files in /boot, mbr and bios"
 maintainer="teldra <teldra@rotce.de>"
 license="Hugware"
 homepage="https://github.com/tastytea/hashboot"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=5520c4e3a83fbeb0121fc8214f23681d6d88289a35f832fa1fbc1684ef4583ad
+archs=noarch
```

Comment:
below revision

Reply:
now travis says, distfiles after archs. my local xlint is telling very different things. o_O